tests/sentry/api/test_event_search.py

```diff
@@ -103,7 +103,7 @@ def test_raw_search_anywhere(self):
             SearchFilter(
                 key=SearchKey(name='message'),
                 operator='=',
-                value=SearchValue(raw_value='hello there'),
+                value=SearchValue(raw_value='hello   there'),
```

Comment:
@saragilford just wanted to highlight this change. We used to remove spaces in between raw search terms. Now we're preserving them. It can be changed back if needed. Just let me know. :)